src/utils/hints/transactionsHints.js

```diff
@@ -0,0 +1,30 @@
+export const transactionsHints = {
+  hash: 'Unique identifier of the transaction.',
+  created: 'Estimated date, time, and block height when the transaction has been sent.',
```

Comment:
This doesn't follow "Block height and estimated date and time of the..." that Michele asked us to use

Reply:
fixed